### ISSUE TITLE
Error Jellyfin - not working

### ISSUE THREAD

**Issue by street5a** [jellyfin/mopidy-jellyfin#98]
Hello, i tried to get jellyfn as a raspi client to work. I am using a server with jellfin for music. My plan was to use a Hifiberry as a highquality soundsource. So the Raspi should be a jellyfin client which i can controll via the jellyfin app on my mobile.

I did install raspian. I did install mopidy. I did install the jellyfin add on. but it is not working.

As soon as i start jellyfin i get the error:

> Blockquote WARNING  Found jellyfin configuration errors, the extension has been automatically disabled:
WARNING    jellyfin/username must be set.
WARNING    jellyfin/hostname must be set.
WARNING  Please fix the extension configuration errors or disable the extensions to silence these messages.

i changed  with

> sudo nano /home/pi/.config/mopidy/mopidy.conf

the . confile to

> [jellyfin]
#hostname = http://192.168.200.XX
#port = 8096
#username = KuXX
#password = XXXx
#libraries = Music
#albumartistsort = True
#album_format = {ProductionYear} - {Name} (Optional: will default to "{Name}" i$
#max_bitrate = 9999999

But if i use via terminal

> mopidy config

i still says

> [jellyfin]
enabled = false  ; Extension disabled due to config errors.
username =   ; Must be set.
user_id = 
password = 
hostname =   ; Must be set.
libraries = 
albumartistsort = 
port = 
client_cert = 
client_key = 
album_format = 
max_bitrate = 
watched_status = false

Do you have any idea what i am doing wrong?

**Comment by mcarlton00** [comment]
It looks like you have a config header for jellyfin set, but all of it's contents are commented out so it looks like an empty service.

Change 
```
[jellyfin]
#hostname = http://192.168.200.XX
#port = 8096
#username = KuXX
#password = XXXx
#libraries = Music
#albumartistsort = True
#album_format = {ProductionYear} - {Name} (Optional: will default to "{Name}" i$
#max_bitrate = 9999999
```
to
```
[jellyfin]
hostname = http://192.168.200.XX
port = 8096
username = KuXX
password = XXXx
libraries = Music
albumartistsort = True
album_format = {ProductionYear} - {Name}
max_bitrate = 9999999
```

Note: Also remove the "Optional:" bit from album_format like I did above.

**Comment by street5a** [comment]
Puh it is working now. Thank you.

There is just one last Problem now. Mopidy is running in the autostart. But Jellyfin is not. Jellyfin is only working, if i open the desktop icon of modipy. In the Terminal which opens is standing, that mopidy already is running. But only then i can start jellyfin from a mobile device. If the system is just starzing, i can not find the raspbery as a client in the jellyfin app.

I tried to put the desktop icon in the autostart. But it is not working. Its relevant because i don' want to huse the raspi with a screen.

**Comment by mcarlton00** [comment]
If you're launching mopidy via systemd or whatever init system you're using, there's a good chance it's running as the `mopidy` user, not your user account.  In this case the config file would be at `/etc/mopidy/mopidy.conf`, not `/home/pi/.config/mopidy/mopidy.conf`.

You may want to take a look at this page if you haven't seen it yet: https://jellyfin.org/docs/general/clients/mopidy.html#raspberry-pi-remote-controlled-speakers

**Comment by street5a** [comment]
I already changed both config files. No difference :/ 

When i am running sudo mopidyctl config
There are errors saying
`Jan 07 17:51:54 raspberrypi mopidy[537]: ERROR    [MainThread] mopidy_jellyfin.http Failed to reach Jellyfin public API on try 0 with problem: `
perhaps this is the problem?

The weird thing about it is just...When i manually click on the mopidy.desktop icon, it is working fine. But the autostart ist not working. It is running mopedy just after the stystem starts. But Jellyfin only by manually clicking on the mopediy icon.


I did try to create autostarts. but this is not working - i am not sure if i'm  doing something wrong.
